scripts/game.js

```diff
@@ -0,0 +1,64 @@
+var rows = 50;
+var cols = 100;
+
+var playing = false;
+
+// define grid using 'new' operator to create an instance of the grid object
+var grid = new Array(rows);
+var nextGrid = new Array(rows);
+
+var timer;
+var reproductionTime = 100;
+
+
+
+function buildGrid() {
+  for (var i = 0; i < rows; i++) {
+    grid[i] = new Array(cols);
+    nextGrid[i] = new Array(cols);
+  }
+}
+function loadGrid() {
```

Comment:
Hmm maybe I'm just not familiar - but I'm not sure why there's this extra layer here?

I think you could get rid of this function and then just call `window.onload = buildGrid` at the bottom?
Or rename `buildGrid` above to `loadGrid` if that semantic naming is important!